config.go

```diff
@@ -124,6 +128,7 @@ func Provide(in ConfigFlagIn) (Config, error) {
 	config.FirmwareName, _ = in.FlagSet.GetString(FirmwareNameKeyName)
 	config.BootTime, _ = in.FlagSet.GetInt64(BootTimeKeyName)
 	config.URL, _ = in.FlagSet.GetString(URLKeyName)
+	config.URL += XMIDTPathURL
```

Comment:
Why couldn't we just pass the `/api/v2/device` suffix as part of the flag?

Reply:
This is to be more like c parodus https://github.com/xmidt-org/parodus/blob/f44fc2c8d9d792e6cea7a0f8bc127b1b8271e2f9/src/config.h#L60